### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by camelid** [rust-lang/cargo-bisect-rustc#117]
**TL;DR** Latest official version cannot bisect past circa May – I would appreciate a new release

In May, **`@alexcrichton`** introduced the embed-bitcode option to Rust. Unfortunately, the latest official version of cargo-bisect-rustc uses the **system cargo** (for me, the latest version of cargo) to invoke rustc, which means that anytime I bisect longer ago than May, I get an error from rustc saying it doesn't recognize the option. @ehuss said in Zulip that PR #98 fixes this, but it would be nice to be able to download an official released version with `cargo install`.

**EDIT:** This should have a major version bump, so it will be 0.6.0, not 0.5.3.

**Comment by Mark-Simulacrum** [comment]
cc @spastorino -- I forget who all was publishing this, but seems like a reasonable thing to issue a new release. I likely will not have a chance to do so for a while though.

**Comment by spastorino** [comment]
@camelid if you can prepare a PR with the changelog I can quickly release a new version :).
